website/content/en/status/report-2023-01-2023-03/clusteradm.adoc

```diff
@@ -31,7 +31,7 @@ This task is in conjunction with the FreeBSD Foundation and donors/sponsors.
 
 ==== FreeBSD Official Mirrors Overview
 
-Current locations are Australia, Brazil, Germany, Japan (two full mirror sites), Malaysia, South Africa, Taiwan, United Kingdom (full mirror site), United States of America (California, New Jersey [the primary site], and Washington).
```

Comment:
Square brackets within parentheses look very weird, to me. Maybe weird to my UK eye – until today, I assumed that square brackets were a US preference (so, it looked like a mixture of Americanised within de-Americanised). 

Today I found <https://english.stackexchange.com/a/11173/11504> –

>> Nesting parentheses should be done with square brackets "([ ])". …

– but still, it looks weird. 

----

Taking <https://english.stackexchange.com/q/11155/11504> and <https://english.stackexchange.com/q/553839/11504> as a whole, including comments, I lean towards _avoidance_ of square brackets (or parentheses) within parentheses.